cluster-agent/controllers/argoproj.io/application_info_cache/application_info_cache.go

```diff
@@ -11,6 +11,20 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/log"
 )
 
+// The cache exists because we do want to overwhelm the controller (or the database) with various operations frequently: most
```

Comment:
```suggestion
// The cache exists because we do not want to overwhelm the controller (or the database) with various operations frequently: most
```

Reply:
:laughing: , whoops, thanks for catching this.